{{ cookiecutter.project_slug }}/Dockerfile

```diff
@@ -7,6 +7,9 @@ WORKDIR /code/
 COPY ./requirements requirements
 RUN pip install -r requirements/dev.txt
 
+# Install gettext in order to use Django's Translation system for internationalization support
+RUN apt-get update && apt-get install gettext -y
```

Comment:
This'll have to be in `Dockerfile.prod` as well.

Reply:
Understood. Let me add that right now.